core/converter/min-max-converter.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @module montage/core/converter/min-max-converter
+ * @requires montage/core/converter/converter
+ */
+var Converter = require("./converter").Converter;
+
+function _clamp(value) {
+
+    return value <= this.min ? this.min : value >= this.max ? this.max : value;
+    
+};
+
+/**
+ * Converts a number to an integer
+ * @class MinMaxConverter
+ * @extends Converter
+ */
+var MinMaxConverter = exports.MinMaxConverter = Converter.specialize({
+
+    constructor: {
+        value: function MinMaxConverter (min, max) {
```

Comment:
Please remove the passing of arguments in constructor, the whole constructor is unecessary